Networking/Networking/Model/StoredOrderSettings.swift

```diff
@@ -0,0 +1,49 @@
+import Foundation
+import Codegen
+
+/// Models a pair of `siteID` and Order Settings
+/// These entities will be serialised to a plist file using `OrdersModuleSettings`
+///
+public struct StoredOrderSettings: Codable, Equatable {
+
+    public struct Setting: Codable, Equatable {
+        public let siteID: Int64
+        public let orderStatusesFilter: [OrderStatusEnum]?
+        public let dateRangeFilter: OrderDateRangeFilter?
+
+        public init(siteID: Int64,
+                    orderStatusesFilter: [OrderStatusEnum]?,
+                    dateRangeFilter: OrderDateRangeFilter?) {
+            self.siteID = siteID
+            self.orderStatusesFilter = orderStatusesFilter
+            self.dateRangeFilter = dateRangeFilter
+        }
+
+        public func numberOfActiveFilters() -> Int {
+            var total = 0
+            if let orderStatusesFilter = orderStatusesFilter, orderStatusesFilter.count > 0 {
+                total += 1
+            }
+            if let dateRangeFilter = dateRangeFilter, dateRangeFilter.filter != .any {
+                total += 1
+            }
+
+            return total
+        }
+
+
+
+        enum CodingKeys: String, CodingKey {
+            case siteID = "site_id"
+            case orderStatusesFilter = "order_statuses_filter"
+            case dateRangeFilter = "date_range_filter"
```

Comment:
In theory yes. In practice I didn't used it because the encoder is not a strict part of the model, so it depends where you use the model. You can implement basically two different encoders, and since I'd like to have always the same result for these type of models stored in our app, if you use one Encoder with the `KeyEncodingStrategy` and the other one without the strategy, the result will not be the same. Using instead declared coding key, this problem doesn't exist.